lib/src/ast/event.dart

```diff
@@ -153,20 +156,19 @@ class ParsedEventAst extends TemplateAst
   @override
   int get quotedValueOffset => valueToken?.leftQuote?.offset;
 
-  /// Offset of `(` prefix in `(eventName.postfix)`.
+  /// Offset of `(` prefix in `(eventName.reductions)`.
   @override
   int get prefixOffset => prefixToken.offset;
 
-  /// Offset of `)` suffix in `(eventName.postfix)`.
+  /// Offset of `)` suffix in `(eventName.reductions)`.
   @override
   int get suffixOffset => suffixToken.offset;
 
-  /// Name `postfix` in `(eventName.postfix)`; may be `null` to have no value.
+  /// Name `filters` in `(eventName.ctrl.shift.a)`; may be empty.
```

Comment:
I'm not sure where 'filters' came from? Perhaps from an older version when you called it filters?

Reply:
Yep, exactly. Fixed!